client/src/app/shared/forms/form-validators/user-validators.service.ts

```diff
@@ -56,6 +57,17 @@ export class UserValidatorsService {
       }
     }
 
+    this.USER_PASSWORD_OPTIONAL = {
```

Comment:
Why don't you use `USER_PASSWORD`?

Reply:
I have to remove the require constraint.